tools/serverpod_cli/lib/src/util/string_manipulation.dart

```diff
@@ -1,17 +1,24 @@
-import 'package:super_string/super_string.dart';
-
 /// Splits a string on the separator token unless the token is inside
-/// brackets or angle brackets, ( ) and < >.
+/// brackets, angle brackets, ( ) and < >, single quotes, or double quotes.
+/// Handles nested brackets and skips separators within single (' ')
+/// or double (" ") quotes.
 List<String> splitIgnoringBrackets(
   String input, {
   String separator = ',',
 }) {
   List<String> result = [];
   StringBuffer current = StringBuffer();
   int depth = 0;
+  bool insideSingleQuotes = false;
+  bool insideDoubleQuotes = false;
+
+  for (int i = 0; i < input.length; i++) {
+    var char = input[i];
 
-  for (var char in input.iterable) {
-    if (char == separator && depth == 0) {
+    if (char == separator &&
+        depth == 0 &&
+        !insideSingleQuotes &&
+        !insideDoubleQuotes) {
```

Comment:
Question: It seems like we have complicated the logic here. Shouldn't it be enough to simply add the same logic for `"` and `'` as applies for `<>` and `()`. That is, just use the depth to understand when we should split the line?